CONTRIBUTING.md

```diff
@@ -300,6 +306,16 @@ OPENAI_API_KEY=sk-XXXXXXXX
 OPENAI_API_BASE_URL=https://XXXXXXXX (Should you utilize an OpenAI proxy service, kindly specify this)
 ```
 
+### Unit & Integration
+As integration tests require token and third-party setup, it is costly to run locally. Ideally, developers should run unit test locally and run integration tests in github/CI/CD. 
```

Comment:
Could we make it explicitly clear about running integration tests in github? Like how to trigger it and how to check the results. I want to make sure that the doc is clear to every contributor no matter what kind of backgrounds they have.
Thanks!

Reply:
> Could we make it explicitly clear about running integration tests in github? Like how to trigger it and how to check the results. I want to make sure that the doc is clear to every contributor no matter what kind of backgrounds they have. Thanks!

sure, made this more clear.